### ISSUE TITLE
Error in Script

### ISSUE THREAD

**Issue by cbjenral** [soonick/poMMo#113]
when i installed the new version through netfirms host these days i founded some coding lines in the home pages appear to me i talked the support and they fixed it but when i try to send a test e-mail 
## these lines appears to me 

" Warning: fsockopen() [function.fsockopen]: unable to connect to www.behealthymow.com:80 (Connection timed out) in /hermes/bosweb25c/b1899/nf.esmael91/public_html/business1/inc/classes/mailctl.php on line 93 

Notice: 1356599403 >>> Error Spawning Page! *\* Errno : Errstr: 110 : Connection timed out in /hermes/bosweb25c/b1899/nf.esmael91/public_html/business1/inc/classes/mailctl.php " 

---

when i talked to the support team they told me that

---

We apologize for any inconvenience this may have caused you. I've enabled allow_url_fopen and allow_url_include for your account but still the issue persists at http://behealthymow.com/business1/admin/mailings/mailings_start.php . 

However, there's no server end issues since a test script that uses fsockopen() function that connects to port 80, is working fine at http://behealthymow.com/business1/admin/test_fsockopen.php . 

Hence, your issue is an application/scripting issue for which we don't provide any support. 
## You need to contact your application vendor/developer for further assistance on the issue.

when i was using the old version a month ago it was working great. now it stopped

**Comment by soonick** [comment]
Have checked if the old version of pommo still works in your server?. It would be good to try that to discard a change in the server causing the problem.

**Comment by frogabog** [comment]
I had a problem with this kind of path with MODX on a site a few months back.  Things would work one day, and not the next.  Three times this occurred in less than 3 months.  The only constant was this path, similar each time, but yet slightly different.  Your post caught my eye because the hermes/bobsweb/blahblah... path is the same path as the box this particular site was on.  Turns out that path changes, arbitrarily.  And according to the host, they could not inform me of when this path might change.  

I ended up moving the site from this host.  I hope this helps you narrow down the problem on your end. Sounds like this old host and your current host are buying server space on the same boxes.
